### PR TITLE
Easier way to automatically add organisation data into CID

### PR DESCRIPTION
**Pull request by cristian-popa**
*Issue #, if available:*

*Description of changes:* Make it easier to include all this rich organisation data by default into our CID datasets

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by awssteph — APPROVED** [review]

File: content/Cost/300_Labs/300_Organization_Data_CUR_Connection/5_Join_Cost_Intelligence_Dashboard.md [hunk 1]
```diff
@@ -1,68 +1,43 @@
...
-This section is **optional** and shows how you can add your AWS Organization Data to your **Enterprise Dashboards** - [200_Enterprise_Dashboards]({{< ref "/Cost/200_Labs/200_Enterprise_Dashboards" >}}).
+This section is **optional** and shows how you can add your AWS Organization Data to your **Cloud Intelligence Dashboards** - [Cloud_Intelligence_Dashboards]({{< ref "/Cost/200_Labs/200_Cloud_Intelligence" >}}).
 
-This example will show you how to map your Enterprise Dashboard linked_account_id to your Organizations account_number to add account information that is meaningful to your organization.
 This is to replace this step: https://wellarchitectedlabs.com/cost/200_labs/200_enterprise_dashboards/2_modify_cost_intelligence/. 
 
-1. Go to the **Amazon QuickSight** service homepage
 
-2. In **QuickSight**, select the **summary_view** Data Set
+1. Login via SSO in your Cost Optimization account, go into the Athena console:
```
Comment (awsdunc) [inline content/Cost/300_Labs/300_Organization_Data_CUR_Connection/5_Join_Cost_Intelligence_Dashboard.md:17]: Will every customer using this have AWS SSO configured?

File: content/Cost/300_Labs/300_Organization_Data_CUR_Connection/5_Join_Cost_Intelligence_Dashboard.md [hunk 2]
```diff
@@ -1,68 +1,43 @@
...
 
-3. Select **Edit data set**
+2. Modify and run the following query to override the account_map view so that it includes the enhanced organisation data. Make sure you replace the correct cur database and if your organisation data resides in a separate athena database, ensure you pick the correct database for your organisation data table. Also if you added tags to the organisation data ensure you add your custom tags as well, in this example we have a team tag and an environment tag in the SQL query.
```
Comment (awsdunc) [inline content/Cost/300_Labs/300_Organization_Data_CUR_Connection/5_Join_Cost_Intelligence_Dashboard.md:19]: Athena

Comment (awsdunc) [inline content/Cost/300_Labs/300_Organization_Data_CUR_Connection/5_Join_Cost_Intelligence_Dashboard.md:19]: There is a lot going on in this paragraph. Can it be broken down to be clearer?

File: content/Cost/300_Labs/300_Organization_Data_CUR_Connection/5_Join_Cost_Intelligence_Dashboard.md [hunk 3]
```diff
@@ -1,68 +1,43 @@
...
-![Images/Org_Data.png](/Cost/300_Organization_Data_CUR_Connection/Images/Org_Data.png)
 
-6. Select the **two circles** to open the join configuration then select **Left** to change your join type:
-![Images/dashboard_mapping_6.png](/Cost/300_Organization_Data_CUR_Connection/Images/dashboard_mapping_6.png)
+3. Navigate to the Quicksight console, select datasets, save all datasets one by one(this will also trigger a refresh) 
 
-7. Create following **join clause** :
-	- **linked_account_id** = **id**
-Click **Apply**
+4. Now your Cloud Intelligence dashboards should have the correct mapping of account name to account numbers and if you're trying to enhance your dashboard via custom analises you should be able to enrich your dashboards to display and slice and dice the additional fields included such as parent OU, account status, joined method and tags.
```
Comment (awsdunc) [inline content/Cost/300_Labs/300_Organization_Data_CUR_Connection/5_Join_Cost_Intelligence_Dashboard.md:41]: analysis?

Comment (awsdunc) [inline content/Cost/300_Labs/300_Organization_Data_CUR_Connection/5_Join_Cost_Intelligence_Dashboard.md:41]: This sentence is too long, please break it after 'account numbers'. 'If you're trying..'

**Comment by iakov-aws** [comment]
Makes total sense added here with minor fixes:
https://github.com/awslabs/aws-well-architected-labs/pull/836

**Comment by iakov-aws** [comment]
All comments are taken into account here: https://github.com/awslabs/aws-well-architected-labs/pull/837/files

**Pull request closed without merging**